### ISSUE TITLE
props

### ISSUE THREAD

**Issue by tireddadwhogames** [SqueegeeDino/XLObjectDropper#25]
Once you place the "worklight small" it does not let you reselect it to edit it or move it at all

**Comment by MCBTay** [comment]
Good catch!  looks like the small work light is missing collision.  Will get this updated and pushed out soon.
